tests/fixtures/example_transformations/transform_content/count_content_values/input.schemapack.yaml

```diff
@@ -0,0 +1,69 @@
+# a simple schemapack:
+schemapack: 0.3.0
+classes:
+  Dataset:
+    id:
+      propertyName: alias
+    content:
+      {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "additionalProperties": false,
+        "description": "A dataset that is a collection of files.",
+        "properties":
+          {
+            "dac_contact": { "type": "string" },
+            "samples_summary":
+              {
+                "type": "object",
+                "additionalProperties": false,
+                "properties":
+                  {
+                    "stats": { "type": "object", "additionalProperties": true },
+                  },
+                "required": ["stats"],
+              },
+          },
+        "required": ["samples_summary"],
```

Comment:
This reminds me the previous implementation, where we assumed there would already be a prior `add_content_property` transformation and `count_content_values` was built on top of those added properties. I think now we do not have such a coupling. 

Of course, this is a valid example, but in principle, the transformation is expected to add the entire `samples_summary`, not only the `description` under `stats`.